designer/server/src/test/scala/pl/touk/nussknacker/test/base/it/NuResourcesTest.scala

```diff
@@ -107,7 +109,12 @@ trait NuResourcesTest
 
   protected val processChangeListener = new TestProcessChangeListener()
 
-  protected lazy val deploymentManager: MockDeploymentManager = MockDeploymentManager.create()
+  protected lazy val deploymentManager: MockDeploymentManager = MockDeploymentManager.create(
+    // In pekko-based resources tests, it is hard to use deploymentManager.withStubbedDeployResult(...) syntax because checks are done in function pipeline.
+    // Because of that we stub results with happy-path defaults. After rewriting endpoint to Tapir and tests to rest-assured-based it should be easier
+    defaultDeployResult = Future.successful(None),
```

Comment:
This strange defaults were hardcoded, I tried to remove them but as you can see in comment, it is not so easy.